src/lib/y2storage/planned/has_size.rb

```diff
@@ -160,10 +160,10 @@ def device_extra_size(device, total_size, total_weight, assigned_size, rounding)
           extra_size = extra_size.ceil(rounding)
           extra_size = available_size.floor(rounding) if extra_size > available_size
 
-          new_size = extra_size + device.size
+          new_size = device.size + extra_size
```

Comment:
what is reason for such change? I hope you do not overwrite plus to not be commutative.